scored.py

```diff
@@ -8,113 +8,79 @@
 if os.path.exists(os.getcwd()+'/issuelist.txt'):
 	os.remove(os.getcwd()+'/issuelist.txt')
 
-
-class scrapeAMSJournal(object):
-	def __init__(self, url):
+'''Purpose: To create seedlist of journal issues and extract article page metadata from
+			AMS and other journal websites 
+	Inputs: Instantiate with URL of website, 0 or 1 (indicating xpath or classtag) and file of xpaths
+			or class tag string
+'''
+
+class scored(object):
+	def __init__(self, url, num, input):
```

Comment:
@mich326  @kushaank and @shashank1994 please be mindful of Python's builtin function and special keywords. input is one of these - See https://docs.python.org/2.7/library/functions.html for ref. 
Also, check your editor to see how to keep track of these.